packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -161,9 +162,12 @@ sub-update-confirm-year = { $intervalCount ->
   *[other] I authorize { -brand-name-mozilla }, maker of { -brand-name-firefox } products, to charge my payment method <strong>{ $amount } every { $intervalCount } years</strong>, according to payment terms, until I cancel my subscription.
 }
 
-sub-update-submit = Change Plans
```

Comment:
how strict are we right now about changing string ids when we change the string meaning? Are we still `en` only?

Reply:
Yeah, I don't think we have this hooked up to Pontoon for real localization yet